### PR TITLE
BONNIE-326

### PR DESCRIPTION
**Pull request by holmesie**
Addresses BONNIE-326

JIRA Test: https://jira.mitre.org/browse/BONNIE-917

Related Pull Requests:
https://github.com/projectcypress/health-data-standards/pull/518
https://github.com/projecttacoma/cql_qdm_patientapi/pull/53

### CODE REVIEW

File: lib/models/cql_measure.rb [hunk 1]
```diff
@@ -97,6 +95,7 @@ def all_data_criteria
   end
 
   # Note whether or not the measure is a continuous variable measure.
+  before_save :set_continuous_variable
```
Comment (hossenlopp) [inline lib/models/cql_measure.rb:98]: I don't think the move of this before_save is relevant for fixing this issue. But I think it does make a lot of sense.

**Review by hossenlopp — APPROVED** [review]

**Comment by hossenlopp** [comment]
I still need to run the test.

**Comment by lukoktonos** [comment]
Regression test passed.

**Comment by hossenlopp** [comment]
Manual JIRA test didn't quite pass, needs slight reworking. But otherwise looks good. Comments are in the execution here: https://jira.mitre.org/secure/enav/#/436

**Review by lukoktonos — APPROVED** [review]
Passed regression test.

**Comment by hossenlopp** [comment]
Manual JIRA test has been amended and passes.

**Review by c-monkey — APPROVED** [review]
Test https://jira.mitre.org/browse/BONNIE-917 passed.
https://oncprojectracking.healthit.gov/support/browse/BONNIE-277 is resolved.
https://oncprojectracking.healthit.gov/support/browse/BONNIE-326 is resolved.

**Pull request merged**